src/views/onboarding/collection-view/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function OnboardingCollectionView({
 	return (
 		<>
 			<HashiHead>
-				<meta name="robots" content="noindex" />
+				<meta name="robots" content="noindex, nofollow" />
```

Comment:
Fast follow fist for the onboarding work. lmk if I should separate this into a separate PR.